### PR TITLE
chore(node): allow current node version

### PR DESCRIPTION
**Pull request by vicb**
& some cleanup

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
```
Comment (IgorMinar) [inline .travis.yml]: please separate the cleanup from node version upgrade.

Comment (IgorMinar) [inline .travis.yml]: please also update .nvmrc file

**Review by IgorMinar — CHANGES_REQUESTED** [review]
please also update .nvmrc

File: .travis.yml (hunk 1 above)
Comment (vicb) [inline .travis.yml]: done

Comment (vicb) [inline .travis.yml]: done

**Review by vicb — COMMENTED** [review]
fghfgh

**Review by IgorMinar — APPROVED** [review]

**Comment by alexeagle** [comment]
merged in 0621f07

**Pull request closed without merging**